package.json

```diff
@@ -11,17 +11,19 @@
     "release": "pnpm run build && changeset publish",
     "build": "turbo run build --filter=\"@astrojs/*\"",
     "build:ci": "turbo run build:ci --filter=\"@astrojs/*\"",
-    "format:ci": "pnpm run format",
+    "ci": "pnpm run biome:ci && pnpm run format:prettier --check && pnpm run lint",
```

Comment:
I added a `ci` command, to run during the CI, which runs linting and formatting in **check** mode

Reply:
I think you explained that to me before, but I forgot.. What is **check** mode again?

Reply:
`check` mode is `format` + `lint`